### ISSUE TITLE
Text alignment in email verification page 

### ISSUE THREAD

**Issue by saurabhkanswal** [anitab-org/portal#529]
## Description
As a user, when i redirect to email-verification page after clicking on signup button. 
then the text is not properly align in email verification page, which look a little bit odd.

## Mocks
![1](https://user-images.githubusercontent.com/26677033/82113743-01c76000-9776-11ea-86c3-f1722d31f723.PNG)

### Update [Required]
- [ ] align the text properly and comment with screenshot before creating a pull request.

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
 30 minutes

**Comment by LaibaBasit008** [comment]
Hi I would Like to work on this issue

**Comment by SanketDG** [comment]
@LaibaBasit008 there you go!

**Comment by LaibaBasit008** [comment]
[Screenshot](https://drive.google.com/file/d/1V7L0-bDAgK1g2bsyYJTgiyOPME_1ymGa/view?usp=sharing)

**Comment by khushishikhu** [comment]
may i work on this issue?

**Comment by khushishikhu** [comment]
if no one is working on this issue may i work on this??

**Comment by sakshi1499** [comment]
Sure @khushishikhu go ahead!

**Comment by saideepesh000** [comment]
@khushishikhu, If you did set up the project correctly then please help me, When I type "..manage.py migrate", I am getting this error "function 'OGR_F_GetFieldAsInteger64' not found".

**Comment by khushishikhu** [comment]
@saideepesh000  actually i am also a new learner and i am not getting such kind of error.  sorry i don't know why are you getting this error. thanks..

**Comment by saideepesh000** [comment]
> 
> 
> @saideepesh000 actually i am also a new learner and i am not getting such kind of error. sorry i don't know why are you getting this error. thanks..

@khushishikhu, did you follow the steps which were on README.md, exactly, and it worked

**Comment by khushishikhu** [comment]
> > @saideepesh000 actually i am also a new learner and i am not getting such kind of error. sorry i don't know why are you getting this error. thanks..
> 
> @khushishikhu, did you follow the steps which were on README.md, exactly, and it worked

yes just struck on migrate step and it's because some of the dependencies i haven't installed yet
